src/ILICheck.Web/PhysicalFileProvider.cs

```diff
@@ -11,19 +13,26 @@ namespace ILICheck.Web
     /// </summary>
     public class PhysicalFileProvider : IFileProvider
     {
+        private readonly IConfiguration configuration;
+        private readonly string rootDirectoryEnvironmentKey;
+
         private bool initialized;
 
         /// <inheritdoc/>
         public DirectoryInfo HomeDirectory { get; private set; }
 
+        /// <inheritdoc/>
+        public string HomeDirectoryPathFormat { get; private set; }
+
         /// <summary>
-        /// Initializes a new instance of the <see cref="PhysicalFileProvider"/>
-        /// at the given <paramref name="root"/> directory path.
+        /// Initializes a new instance of the <see cref="PhysicalFileProvider"/> at the given root directory path.
         /// </summary>
-        /// <param name="root">The root directory. This must be an absolute path.</param>
-        public PhysicalFileProvider(string root)
+        /// <param name="configuration">The configuration.</param>
+        /// <param name="variable">The name of the environment variable containing the root directory path.</param>
+        public PhysicalFileProvider(IConfiguration configuration, string variable)
```

Comment:
PP: `variable` zu etwas aussagekräftigerem umbenennen
```suggestion
        public PhysicalFileProvider(IConfiguration configuration, string rootDirectoryEnvironmentKey)
```